processing/data_collection/gazette/spiders/rj_sao_goncalo.py

```diff
@@ -0,0 +1,28 @@
+from gazette.items import Gazette
+import datetime
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjSaoGoncaloSpider(BaseGazetteSpider):
+    TERRITORY_ID = '3304904'
+    name = 'rj_sao_goncalo'
+    allowed_domains = ['www.pmsg.rj.gov.br']
+    download_gazette_url = 'http://www.pmsg.rj.gov.br/diario/{}.pdf'
+    start_urls = ['http://www.pmsg.rj.gov.br/diario_oficial.php']
+
+    def parse(self, response):
+        date = datetime.date.today()
+        end_date = datetime.date(2015, 1, 1)
```

Comment:
Change this date to the first document available date (`03/02/1998`, I think) and let the DropItem (In the pipeline) take care of the period of interest (That can be mutable so let the pipeline controls it helps with the maintenance)